### ISSUE TITLE
Flash has auto styling that interfere with SVG elements

### ISSUE THREAD

**Issue by liranelisha** [primer/react#3020]
### Description

I am writing to report an issue with a Flash component that I have been using on my website. The component appears to have an auto-styling feature that is interfering with the SVG elements on my web pages.

Specifically, the auto-styling feature is causing the SVG elements to be displayed incorrectly with fixed marginRight of 2 (8px). This is an issue, as the SVG elements are a part of the design and functionality of my pages.

I have tried adjusting the CSS styles for the SVG elements to no avail, as the auto-styling seems to get a priority. The only way to resolve the issue seems to be to add the `!important`.

I would greatly appreciate it if you could look into this issue and provide a solution as soon as possible. If necessary, I can provide you with access to my website and the code for the Flash component and the SVG elements.

Thank you for your prompt attention to this matter.

![image](https://user-images.githubusercontent.com/76681072/224741082-c1b93cf7-ad0e-4c2a-bfab-66884253ef09.png)

### Steps to reproduce

create a `StyledOcticon` inside a `Flash` component

### Version

35.21.0

**Comment by dejnon** [comment]
Here is another reproduction:
<img width="993" alt="image" src="https://user-images.githubusercontent.com/873549/224742754-adaf5853-807f-4064-bc34-d7956f8f4ae5.png">

**Comment by liranelisha** [comment]
this issue was seen in the devportal
example: https://devportal.githubapp.com/devportal/apps/github?tab=deployments
when we have the "external link" icon when the text include a link to external source

**Comment by lesliecdubs** [comment]
Thanks for the report, this does seem to be a bug. 

Internal note: I am removing the `react` label from this issue for now so that Primer Design can hopefully prioritize this. @tallys let us know if that's not doable.

**Comment by tallys** [comment]
@lukasoppermann let's take a look at this as part of our work on flash/banner

**Comment by lukasoppermann** [comment]
Yes. 👍

**Comment by liranelisha** [comment]
sill happening...

**Comment by lukasoppermann** [comment]
@liranelisha we will keep this in mind for the new component.

**Comment by lukasoppermann** [comment]
@liranelisha could you verify that the issue is fixed in the new component?

https://primer.style/components/banner/react/draft

**Comment by liranelisha** [comment]
i think we are currently unable to upgrade to version 36.27.0 as it cause some unit testing fails on our side. but i'll keep on my list to test it and reopen if needed once upgraded

**Comment by TylerJDev** [comment]
Hey @liranelisha! 👋 

Just to confirm, does using `!important` unblock you here? I took a look at the issue, and there isn't a great fix here as the intention was that there'd only be one icon present in Flash at a given time, with that icon being the very first inside the `Flash`. While not ideal, I think the `!important` might be the best way to go here 🤔 

We're eventually planning on deprecating the `Flash` component and suggest that consumers utilize the [`Banner`](https://primer.style/components/banner/react/alpha) component instead. If you need help with debugging any issues with upgrading Primer in your project, definitely feel free to create another issue, or leave a message for us in [#primer](https://github-grid.enterprise.slack.com/archives/CSGAVNZ19)!

I'm going to close this issue for now, but happy to continue the conversation here or to reopen the issue!
